maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -518,6 +518,38 @@ class AndroidDriver(
         )
     }
 
+    override fun setPermissions(appId: String, permissions: Map<String, String>) {
+        val mutable = permissions.toMutableMap()
```

Comment:
Launch app fails due to this, creating a fix for this. Let's defer this PR until then